src/main/java/rs/hybridit/serviceImpl/UserServiceImpl.java

```diff
@@ -0,0 +1,49 @@
+package rs.hybridit.serviceImpl;
+
+import java.util.List;
+import java.util.Optional;
+import org.springframework.stereotype.Service;
+import rs.hybridit.model.User;
+import rs.hybridit.repository.UserRepository;
+import rs.hybridit.service.UserService;
+
+@Service
+public class UserServiceImpl implements UserService {
+
+	private final UserRepository userRepository;
+
+	public UserServiceImpl(UserRepository userRepository) {
+		this.userRepository = userRepository;
+	}
+
+	@Override
+	public User findById(long id) {
+		Optional<User> userOptional = userRepository.findById(id);
+		if (userOptional.isPresent()) {
```

Comment:
this can be simplified to
`return userRepository.findById(id).orElse(null)`

https://www.nurkiewicz.com/2013/08/optional-in-java-8-cheat-sheet.html